Cubical/Structures/Macro.agda

```diff
@@ -0,0 +1,71 @@
+{-# OPTIONS --cubical --no-exact-split --safe #-}
+module Cubical.Structures.Macro where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Function
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.SIP renaming (SNS-PathP to SNS)
+open import Cubical.Functions.FunExtEquiv
+open import Cubical.Data.Sigma
+
+open import Cubical.Structures.Constant
+open import Cubical.Structures.Pointed
+open import Cubical.Structures.NAryOp
+open import Cubical.Structures.Parameterized
+
+data Desc (ℓ : Level) : Typeω where
```

Comment:
It's annoying that this is in `Typeω`... But not much to do about it I guess?

Reply:
Some comments explaining the various field would be helpful. Especially the latter ones

Reply:
I added some comments.